### PR TITLE
ignore pywikibot settings but not itself

### PR DESCRIPTION
**Pull request by lokal-profil**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 
-.gitignore
 lib64
 pip-selfcheck.json
 pywikibot.lwp
 throttle.ctrl
 user-password.py
+user-config.py
 pyvenv.cfg
 *.yaml
```
Comment (sebastian-berlin-wmse) [inline .gitignore]: Remove this duplicated line while you're at it.

**Review by sebastian-berlin-wmse — CHANGES_REQUESTED** [review]
Looks good. Found a related thing to fix.

File: .gitignore (hunk 1 above)
Comment (sebastian-berlin-wmse) [reply .gitignore]: To clarify, I meant line the line with `*.yaml` (which is also a bit further up), which isn't obvious the way Github displays it.

Comment (lokal-profil) [reply .gitignore]: > To clarify, I meant line the line with `*.yaml` (which is also a bit further up), which isn't obvious the way Github displays it.

Thanks. Couldn't figure out which line was meant based on the Github display

**Review by sebastian-berlin-wmse — APPROVED** [review]

**Pull request merged**